website/docs/components/carousel/carousel-code.md

```diff
@@ -33,6 +33,16 @@ tabs: Design('carousel'), A11y('carousel-a11y'), API('carousel-api'), Example('c
 
 :::
 
+## Carousel with focusable Indicators only
```

Comment:
Does this carousel really contains focusable indicators?

Reply:
no, fixed